packages/he/he.txt

```diff
@@ -0,0 +1,2 @@
+שלום
```

Comment:
the dictionary only seems to include two words, was that the intent?
Maybe use a hunspell dictionary instead of a word list.

Reply:
I'm trying to build with a hunspell dictionary, but It's running for a day now. Can you tell me what I'm doing wrong? I got the dictionary I'm trying to build from [here](https://github.com/wooorm/dictionaries/tree/master/dictionaries/he)